src/tests/CoreMangLib/system/runtime/interopservices/marshal/marshalsizeof1.cs

```diff
@@ -480,7 +480,7 @@ public bool NegTest3()
         try
         {
             Object obj = new Object();
-            int size = Marshal.SizeOf(obj);
+            int size = Marshal.SizeOf<object>(obj);
```

Comment:
For posterity, the rest of SizeOf calls here go to the generic method automatically, but for System.Object, the compiler picks the non-generic AOT-warning overload and it doesn't work at runtime.